pkg/devfile/versions/common/projects.go

```diff
@@ -0,0 +1,46 @@
+package common
+
+import "fmt"
+
+// isProjectTypeSupported checks if the given project type is supported in ODO
+func isProjectTypeSupported(typ DevfileProjectsType) bool {
+	for _, supportedType := range SupportedDevfileProjectTypes {
```

Comment:
can we use the `In` function here? 
https://github.com/openshift/odo/blob/adf3006360abc5cb707d9a9f442669989272f4e5/pkg/util/util.go#L80

Reply:
Not really, to use `In()` we may need to convert/typecast every element of `SupportedDevfileProjectTypes` of custom type `DevfileProjectType` to string.